bundles/ranvier-commands/commands/get.js

```diff
@@ -63,6 +63,7 @@ module.exports = (srcPath) => {
       player.addItem(item);
 
       Broadcast.sayAt(player, `<green>You receive loot: </green>${item.display}<green>.</green>`);
+      Broadcast.sayAt(room, `<green>${player.name} receives loot: </green>${item.display}<green>.</green>`)
```

Comment:
So basically, too much annoyance potential?
I wonder if there's an in-between to prevent that. I'd want to know so that I don't try to pick it up and am surprised when it is not there. Maybe we could debounce it somehow.